x-pack/legacy/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/StickyErrorProperties.tsx

```diff
@@ -41,17 +41,9 @@ function TransactionLink({
   }
 
   return (
-    <APMLink
-      path={`/services/${transaction.service.name}/transactions/view`}
-      query={{
-        transactionId: transaction.transaction.id,
-        traceId: transaction.trace.id,
-        transactionName: transaction.transaction.name,
-        transactionType: transaction.transaction.type
-      }}
-    >
+    <TransactionLink transaction={transaction}>
```

Comment:
`TransactionLink` already contain the routing logic. It just takes a `transaction`.